src/main/java/Baymax.java

```diff
@@ -0,0 +1,114 @@
+
+        Scanner in = new Scanner(System.in);
+        String command;
+        String taskDescrip;
+        String[] taskWord;
+        TaskManager tManager = new TaskManager();
+        String horiLine = "____________________________________________________________\n";
+
+        //load record
+        try {
+            tManager.initialiseNewFile();
+        } catch (FileNotFoundException e) {
+            System.out.println("File not found.");
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+
+        tManager.welcome(); //greating
+
+        command = in.nextLine();
+        while (!command.equals("bye")) {
+
+            String[] word_split;
+            word_split = command.split(" ", 2);
+            String taskName = word_split[0];
+
+            //different functions based on command lines
+            switch (taskName) {
+                case "todo":
+                    tManager.addTask(new Todo(word_split[1]));
+                    break;
+                case "deadline":
+                    try {
+                        taskWord = word_split[1].split("/by ", 2);
+                        taskDescrip = taskWord[0];
+                        String ddl = taskWord[1];
+                        tManager.addTask(new Deadline(taskDescrip, ddl));
+                    } catch (ArrayIndexOutOfBoundsException e) {
+                        System.out.println(" ☹ OOPS!!! The description of a deadline cannot be empty.");
+                    } catch (BaymaxException b){
+                        System.out.println( b.getMessage() +" ☹ OOPS!!! Let's do it again.");
+                    }
+                    break;
+                case "event":
+                    try {
+                        taskWord = word_split[1].split("/at ", 2);
+                        taskDescrip = taskWord[0];
+                        String eventTime = taskWord[1];
+                        tManager.addTask(new Event(taskDescrip, eventTime));
+                    } catch (ArrayIndexOutOfBoundsException e) {
+                        System.out.println(" ☹ OOPS!!! The description of an event cannot be empty.");
+                    } catch (BaymaxException b){
+                        System.out.println( b.getMessage() +" ☹ OOPS!!! Let's do it again.");
+                    }
+                    break;
+                case "list":
+                    tManager.printTaskList();
+                    break;
+                case "mark":
+                    System.out.println(horiLine);
+                    try {
+                        tManager.markTask(Integer.parseInt(word_split[1]) - 1);
+                    } catch (ArrayIndexOutOfBoundsException e) {
+                        System.out.println("Please input an integer for task index.");
+                    } catch (BaymaxException b){
+                        System.out.println( b.getMessage() +" ☹ OOPS!!! Let's do it again.");
+                    }
```

Comment:
I believe this would cause an error if you type in any non-numeric input for your second option. `mark abc` would throw a `NumberFormatException` if not handled correctly.